Gemfile

```
@@ -12,6 +12,7 @@ gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rail
 gem 'jbuilder', '~> 2.7' # https://rubygems.org/gems/jbuilder
 gem 'pg', '~> 1.1' # https://rubygems.org/gems/pg
 gem 'puma', '~> 5.0' # https://rubygems.org/gems/puma
+gem 'discard', '~> 1.2'
```

Muy bien, solo un detalle para continuar el orden ya diseñado en el Gemfile:
Deberias poner la linea `gem 'discard', '~> 1.2'` en la linea 11 (debajo de bootsnap) para seguir un orden alfabético y en siguiente formato:

```suggestion
gem 'discard', '~> 1.2' # https://rubygems.org/gems/discard
```

---

Hecho